### DISCUSSION TITLE
How to add Serenity Bdd Report in Jenkins for Webdriverio Mocha project

### DISCUSSION THREAD

**Discussion by GitSiva1991** [serenity-js/serenity-js#1639] [How to... ?]
Hi,
We are able to generate the "Serenity Bdd Report" locally for **WebdriverIO mocha automation project**.
But, how we can add & see the "Serenity Bdd Report" Under "Artifacts" in Jenkins Pipeline. is it possible?

Our objective is, whenever we raise any PR, it will show the result as "pass or fail". along with that,
can we see the serenity bdd report under artifacts?
![image](https://user-images.githubusercontent.com/63959785/210543366-50c6840e-7287-421a-94a0-805ee33829f6.png)

and another doubt is:
Serenity Bdd Report in Jenkins means,
The Report is like in the below screenshot only:
![image](https://user-images.githubusercontent.com/63959785/210543662-bae1556f-90de-480f-9cc2-07a6217a2765.png)

**Comment by jan-molak** [comment]
Hello @GitSiva1991! Serenity BDD produces an HTML report, which you can publish in Jenkins using the [HTML Publisher](https://plugins.jenkins.io/htmlpublisher/) plugin.

You'll need to configure it to publish `target/site/serenity`.

Hope this helps!

**Comment by GitSiva1991** [comment]
Hi @jan-molak, Thanks for the response.
Installed the "HTML Publisher" plugin and serenity bdd report in Jenkins.
getting like this:
![image](https://user-images.githubusercontent.com/63959785/210557056-f5f631d2-5d43-4cb1-a339-677ebcdd1fc6.png)

the report will show as in the above screenshot only?

 it won't show like when we execute in a local environment like the below screenshot:
![image](https://user-images.githubusercontent.com/63959785/210557381-a0023647-3946-449b-810b-638a3ecaed6a.png)

**Comment by jan-molak** [comment]
Hey @GitSiva1991 - I'm not sure the first screenshot is what you intended to attach?

The way Jenkins HTML Publisher plugin works is that it makes any static HTML website (e.g. a Serenity BDD report) available via its embedded web server.
